trollmoves/tests/test_client.py

```diff
@@ -917,6 +934,37 @@ def test_request_push_single_call(send_ack, send_request, clean_ongoing_transfer
     assert len(file_cache) == 1
 
 
+@patch('trollmoves.client.ongoing_transfers', new_callable=dict)
+@patch('trollmoves.client.file_cache', new_callable=deque)
+@patch('trollmoves.client.clean_ongoing_transfer')
+@patch('trollmoves.client.send_request')
+@patch('trollmoves.client.send_ack')
+def test_request_push_backup_targets(send_ack, send_request, clean_ongoing_transfer, file_cache, ongoing_transfers):
+    """Test trollmoves.client.request_push() with a single file."""
+    from trollmoves.client import request_push
+    from tempfile import gettempdir
+
+    MSG_FILE_BACKUP_TARGETS = MSG_FILE2
```

Comment:
```suggestion
    msg_file_backup_targets = MSG_FILE2
```
This should be lower case, and adjusted accordingly also later in the code.

Reply:
Hm, ok. I was just following the syntax used in the rest of the similar setup.